main.py

```diff
@@ -214,7 +211,8 @@ def main(collect: bool = False, resume: Optional[Path] = None):
         parameters['uploads'] = FileManager.file_paths_to_dict(resume, plain_text_resume_file)
         parameters['outputFileDirectory'] = output_folder
         parameters['collectMode'] = collect
-        
+        # check if the config is not set as well
```

Comment:
is this done? looks like a TODO to me

Reply:
previous comment forgot to remove, fixed, thanks